api/fastpm/solver.h

```diff
@@ -49,10 +49,10 @@ typedef struct {
 typedef struct {
     size_t nc;
     double boxsize;
-    double omega_m;
-    double hubble_param;
     double alloc_factor;
 
+    FastPMCosmology * cosmology;
```

Comment:
@rainwoodman In relation to previous comment: should we use cosmology[1] here instead?